reference/index.rst

```diff
@@ -0,0 +1,31 @@
+Ouroboros Network Docs
+======================
+
+The network documentation is a work in progress, we have various
+documents in various stages:
+
+-  `Technical report`_ which contains technical documentation of the
+   network stack: mux layer, protocol descriptions and their state
+   machines. Other parts of the document will be merged / reused in the
+   ``Shelley Network Design`` document. Our aim is to join the two
+   docuents into one consistent form. The technical parts are largely
+   finished, though they need to be synced with recent changes in
+   ``ouroboros-network`` repository.
+
+-  `Ouroboros-Network Component Overview`_ - an overview of all the
+   components maintained by the networking team.
+
+-  `Network Implementation Overview`_ This is a short document which
+   just give an overview of the network stack.
+
+-  `Data diffusion and peer discovery`_ a mind map of notions which was
+   compiled to initially organise our knowledge (requires `xmind`_).
+
+.. _Shelly Network Design: https://docs.google.com/document/d/1b0ygneqYp_JSEmmR_dV8Xcznw5MauoNILLHpI5-2STs/edit
+.. _V1.5 for internal review: https://drive.google.com/file/d/1bV466FbLBbszUFfXzOWPN8jUnILGccwQ/view
+.. _V1.5 for external review: https://drive.google.com/file/d/1tTDwEP63r6iqillwjvrMv_xbXHlNMAXr/view
+.. _Technical report: https://github.com/input-output-hk/ouroboros-network-docs/blob/master/technical-report/network.pdf
+.. _Ouroboros-Network Component Overview: https://github.com/input-output-hk/ouroboros-network-docs/wiki/Ouroboros-Network-Components
+.. _Network Implementation Overview: https://docs.google.com/document/d/1qwFnHXgLQArph5FdxfJMrPfn8Y5KhozYU8i5R6gVZbo/edit
+.. _Data diffusion and peer discovery: https://github.com/input-output-hk/ouroboros-network-docs/blob/master/Data%20diffusion%20and%20peer%20discovery.xmind
```

Comment:
Please remove references to both `Network Implementation Overview` and `Data diffusion and peer discover`.  First one is subsumed by `Shelley Network Design` the other one (some parts of it can be merged with t[he `Technical Report`) is our internal document.

Reply:
Issue #2563